engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,20 +92,22 @@ object WorkflowManagerActor {
  * WorkflowOutputs: Returns a `Future[Option[binding.WorkflowOutputs]]` aka `Future[Option[Map[String, WdlValue]]`
  *
  */
-class WorkflowManagerActor(config: Config)
+class WorkflowManagerActor(isServerMode: Boolean, config: Config = WorkflowManagerActor.defaultConfig)
   extends LoggingFSM[WorkflowManagerState, WorkflowManagerData] with CromwellActor {
 
-  def this() = this(WorkflowManagerActor.defaultConfig)
   implicit val actorSystem = context.system
 
   private val logger = Logging(context.system, this)
   private val tag = "WorkflowManagerActor"
 
   private val donePromise = Promise[Unit]()
 
+  val shouldRestartWorkflows = config.getConfig("backend").getBooleanOr("restart-workflow", default = true)
+
   override def preStart() {
     addShutdownHook()
-    restartIncompleteWorkflows()
+    if (isServerMode && shouldRestartWorkflows) { restartIncompleteWorkflows() }
```

Comment:
Change the if/else to just have a `globalDataAccess` on the line above - no need to assign it or anything